beetmoverscript/script.py

```diff
@@ -30,53 +30,37 @@
 async def async_main(context):
     # balrog_manifest is used by a subsequent balrogworker task that points to a beetmoved artifact
```

Comment:
sure. but 'this beetmover artifact' might be ambiguos since there are many artifacts. I'll clean up my comment though.